### PR TITLE
Style header group-chats

### PR DESCRIPTION
**Pull request by cammellos**
Style group chats according to https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=375%3A3842 

Styles group chats according to designs.

### What's missing

Currently no bold on username on chat notifications
Search bar
Group info styling

status: ready

### CODE REVIEW

**Comment by cammellos** [comment]
@errorists I am working on styling group chats https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat?node-id=375%3A3842 
quick question, what should happen if the user clicks on "Decline invitation"? should we just delete the group chat?

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/aS1ct66VQ6V0cio7vSqS8UoG/Chat">Figma</a></strong></div><div>Created with Figma</div></blockquote>

**Comment by errorists** [comment]
@cammellos Yes, delete the group chat and return to home screen.

File: src/status_im/ui/screens/group/views.cljs [hunk 1]
```diff
@@ -12,46 +20,141 @@
...
+  [react/view {:style styles/no-contacts}
+   [react/text
+    {:style styles/no-contact-text}
+    (i18n/label :t/group-chat-no-contacts)]
+   [buttons/secondary-button {:on-press handle-invite-friends-pressed} (i18n/label :t/invite-friends)]])
+
+(defn number-of-participants-disclaimer [number-of-participants-available]
+  [react/view {:style styles/number-of-participants-disclaimer}
+   [react/text (if (> number-of-participants-available
+                      0)
```
Comment (jeluard) [inline src/status_im/ui/screens/group/views.cljs]: pos? maybe?

**Review by jeluard — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

**Comment by churik** [comment]
@cammellos would you mind to rebase it to current develop?
builds are 8 days old. Sorry for delay in testing.

**Comment by cammellos** [comment]
@churik done, sorry for the delay, is building now

**Comment by churik** [comment]
@cammellos so far:

**1. Group chat owner (creator) is counted as 1/10 group chat members.** In design, owner is not counted as 1/10 possible members. Also there is no search bar on New Group Chat screen.
hhhh

**2. Create New Chat button is hidden under keyboard (IOS)**
sdf

**3. Can't scroll on chat name screen (Android 7.0, LG v20)**

groupm
If I try to scroll with open keyboard, I can see only 6 members - others are hidden under keyboard.

**4. "Cancel" button is missing on chat name screen**
cancel

**5. "Next" button is overlapping with checkbox on Members screen**
next

**6. No design changes on "Group info screen"**
6.1 - no amount of chat members
6.2 - "add members"  isn't under "Members"
6.3 - "Delete chat" instead "Delete and Leave"
6.4 - Admin isn't on the first place in members list
groupinfo


**7. Can't add members to group chat - there is no button to proceed**
Steps:
- create groupchat
- tap on 3 dots
- ope Group chat info
- tap on add member
- select members  and try to proceed

**8. No changes in menus**
According to designs: Overflow menu, Member overflow should be changed.
On current PR build they are the same as in nightly

**Comment by cammellos** [comment]
@churik thanks for testing, sorry I should have mentioned which parts were implemented and which were not.

1) Search bar will be implemented separately. For the counting, @errorists we limit the number of users to 10 (creator included), how should we show this to the user when adding/creating a group chat? 0/9 or 1/10?
2,3,4,5,7 I will fix, while 8 and 6 I will probably leave for a separate PR.

thanks!

**Comment by cammellos** [comment]
@churik 
1 is still pending to hear from @errorists 
2) Could you check is still the case? (I have no ios device, but I can check on the simulator, but since a few things have changed it might be fixed
3) should be fixed
4) Will be in a separate pr
5) Should be fixed
6) Will be in  a separate pr
7) Should be fixed
8) Will be in a separate pr

Thanks!

**Comment by errorists** [comment]
@cammellos sorry for taking so long, tough question actually but I propose a 0/9. If we went with a 1/10 then somewhere in the UI we'd have to include the user creating a group to make it clear that he's taking one of the spots.

**Comment by cammellos** [comment]
@errorists no worries, thanks `0/9` sounds good, @churik wait for testing, I will push this change

**Comment by cammellos** [comment]
@churik changed the count of users, should be ready to be tested once the build is ready

**Comment by churik** [comment]
@cammellos thanks for quick fixes!
2) is still the case unfortunately
5) is still reproducible (now last contact is fully overlapped) - video is [here](http://take.ms/2x9fL)
On IOS and Android.
Same on Group Chat screen - button "Create group chat " overlapped last member in the list.

**Comment by churik** [comment]
@cammellos https://github.com/status-im/status-react/pull/7269#issuecomment-457607668 is fixed in the last build, so PR can be moved to `To test` for the final check.
thanks!

**Comment by cammellos** [comment]
@churik awesome, I will do, thanks!

**Comment by churik** [comment]
@cammellos after upgrade from 0.9.32 for all group chats created before upgrade I (as an admin) instead of group chat history see the screen where I invited myself to join group chat:
invi
Not sure that it is expected. Can you confirm please? 

UPD: same behavior on nightly, but here "Decline" option is added. Is it possible to prevent displaying this screen for such cases?

**Comment by yenda** [comment]
If I'm not mistaken the toolbar changes are local to group chat in this PR? I was thinking of doing the toolbar revamp according to the UI components described here https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=794%3A0 

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index">Figma</a></strong></div><div>Created with Figma</div></blockquote>

**Comment by cammellos** [comment]
@yenda not sure I understand, correct me if I am wrong, this changes only group-chats, not any other kind of chats, is that what you were asking?
cheers

**Comment by cammellos** [comment]
@churik it should be fixed now, I have also changed the upgrade behavior so that users won't have to join existing chats.
The behavior is as follow:


If chat has been created in nightly or 0.9.32 (anything before this PR), on upgrade to this PR they won't have to join the chat.
If chat has been created while this PR was running, the chat will have to be joined.

It does not matter who creates the chat (which version), it only matters which version of the app was running when the app was created on the device. (not sure if the explanation is clear).

thanks

**Comment by yenda** [comment]
@cammellos sorry for the confusion I was just curious if you were already on the implementation of top bar components, but it seems that here you just made a local update to fit the group chat design. What I am saying is that the top bar you implemented will soon be the same all accross the app according to designs here https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index?node-id=794%3A0 

I will work on that after discussing with design team tomorrow. I doesn't matter for that PR

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/cb4p8AxLtTF3q1L6JYDnKN15/Index">Figma</a></strong></div><div>Created with Figma</div></blockquote>

**Comment by churik** [comment]
Note: after upgrade you will see system messages that you automatically joined to chat (if it was created on 0.9.32)
Suppose that this is expected.

Thanks @cammellos, great work here!

**Comment by cammellos** [comment]
@churik thanks for testing!

**Pull request merged**